Lib/idlelib/colorizer.py

```diff
@@ -120,12 +120,10 @@ def LoadTagDefs(self):
             "BUILTIN": idleConf.GetHighlight(theme, "builtin"),
             "STRING": idleConf.GetHighlight(theme, "string"),
             "DEFINITION": idleConf.GetHighlight(theme, "definition"),
-            "SYNC": {'background':None,'foreground':None},
-            "TODO": {'background':None,'foreground':None},
+            "SYNC": {'background': None, 'foreground': None},
+            "TODO": {'background': None, 'foreground': None},
             "ERROR": idleConf.GetHighlight(theme, "error"),
-            # The following is used by ReplaceDialog:
-            "hit": idleConf.GetHighlight(theme, "hit"),
-            }
+        }
```

Comment:
See my other comment, regarding `get_tag_colors()`, explaining this.

Reply:
Since this PR requires the removal of the shell tags from the colorizer's `tag_defs`, and the refactoring that entails, removing "hit" which obviously shouldn't be there was a simple and obvious fix. I'll revert it with a comment if you prefer.

(I will mention that this will affect my PR replacing the search/replace dialogs with a toolbar: GH-9855).

Reply:
I left a comment on your toolbar issue.  With the comment added and 'hit' left, we can resolve this for now.